pkg/cli/predict.go

```diff
@@ -376,3 +380,7 @@ func parseInputFlags(inputs []string) (predict.Inputs, error) {
 
 	return predict.NewInputs(keyVals), nil
 }
+
+func addTimeoutFlag(cmd *cobra.Command) {
+	cmd.Flags().Uint32Var(&predictTimeout, "timeout", 5*60, "The timeout for a prediction to wait for a container boot (in seconds).")
```

Comment:
This is a timeout on the setup, not the prediction, right? I think we might want to differentiate the two because we might eventually want both.

Reply:
Changed to `--setup-timeout`